pkgs/development/libraries/amdvlk/default.nix

```diff
@@ -0,0 +1,88 @@
+{ stdenv
+, lib
+, fetchpatch
+, fetchRepoProject
+, cmake
+, ninja
+, patchelf
+, perl
+, pkgconfig
+, python3
+, expat
+, libdrm
+, ncurses
+, openssl
+, wayland
+, xorg
+, zlib
+}:
+
+with lib;
```

Comment:
We should probably not add lib to such large scopes. In this derivation, `lib` also doesn't seem to be used.